azure-pipelines.yml

```diff
@@ -100,26 +100,29 @@ stages:
     # now stuck getting the full gcc toolchain instead of
     # just pulling in gfortran
     - script: |
+        set -xe
         # same version of gfortran as the wheel builds
-        brew install gcc49
-        # manually link critical gfortran libraries
-        ln -s /usr/local/Cellar/gcc@4.9/4.9.4_1/lib/gcc/4.9/libgfortran.3.dylib /usr/local/lib/libgfortran.3.dylib
-        ln -s /usr/local/Cellar/gcc@4.9/4.9.4_1/lib/gcc/4.9/libquadmath.0.dylib /usr/local/lib/libquadmath.0.dylib
+        curl -L https://github.com/fxcoudert/gfortran-for-macOS/releases/download/4.9.0/gfortran-4.9.0-Mavericks.dmg -o gfortran.dmg
```

Comment:
Can we clone that repo to MacPython and remove references directly to it? It does seem like a bit of a security risk - small probability but really large impact if something goes wrong here.

Reply:
Cc @matthew-brett

Reply:
I will change the link to https://github.com/MacPython/gfortran-install/blob/master/archives/gfortran-4.9.0-Mavericks.dmg and add a sha hash for security.

Reply:
This could be viewed as an issue with openblas-libs, we should either bundle the needed shared objects there or statically compile. Our wheels do ship the needed shared objects, with the names mangled to avoid collision.

Reply:
As for security: this workflow is only used in development. The numpy-wheels repo is the one that builds deliverables.